inc/thorvg.h

```diff
@@ -1769,6 +1769,18 @@ class TVG_API Animation
 public:
     ~Animation();
 
+    /**
+     * @brief Rewrite lottie properties through the slot data
+     *
+     * @param[in] slotJson The lottie slot data
+     *
+     * @retval Result::Success When succeed.
+     * @retval Result::InsufficientCondition When the given parameter is invalid.
+     *
+     * @note Experimental API
+     */
+    Result override(const char* slotJson) noexcept;
```

Comment:
This concept is not suitable for general Animation. Please remove the API implementation from this patch. We can consider introducing LottieAnimation instead.